hw03_cruzmendeze.py

```diff
@@ -0,0 +1,141 @@
+#####################################################################################
+#
+# Author: Eric Cruz-Mendez
+# Username: cruzmendeze
+#
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Google Doc Link: https://docs.google.com/document/d/1P5qvQ1eb9AhFPyGV7J0JwgZ8XdeqU9qpbF82LSsu6yM/edit?usp=sharing
+#
+#####################################################################################
+
+import turtle
+turtle.colormode(255)  # Set colormode to RGB value
```

Comment:
This code belongs in `main()`, the first line (since it is the first thing running in this file, currently).

Reply:
I have added it to main()